src/Api.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+async function UserFolderData() {
+  const response = await fetch("https://bootcamp-api.codeit.kr/api/sample/folder" );
```

Comment:
```suggestion
  const API_BASE_URL = 'https://bootcamp-api.codeit.kr'; // 함수밖으로 꺼내 사용하세요.
  const response = await fetch(`${API_BASE_URL}/api/sample/folder` );
```
[중요도: 중간] 이런 형식으로 쓰면 API 선언할 때 오타를 최소화하면서도 불필요하게 적을 것들이 적어질겁니다 ㅎㅎ

Reply:
url 공통적인 부분을 따로 상수로 만들어 사용하면  가독성도 좋은 것 같고 오타를 줄일 수 있어 좋은 것 같습니다. 
감사합니다!
수정했습니다 https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/pull/185/commits/9be977f00b48bf2d8b022cbe74baed059fe1e33b

const folderData = "https://bootcamp-api.codeit.kr/api/sample/folder"
export async function UserFolderData() {
  const response = await fetch(folderData());
  const data = await response.json();
  return data;
}
이런식으로 url전체를 함수 밖에서 변수로 선언해서 사용하는 것은 괜찮을까요?

Reply:
저는 권장하지 않습니다! 그러면 API 하나당 하나의 상수를 선언해서 사용해야하는데 재사용하지 않을 상수를 굳이 선언하는 이유는 없을 것 같아요.